README.md

```diff
@@ -111,6 +112,35 @@ Array is equal not array:
 * [**12.5.9** Logical NOT Operator (`!`)](https://www.ecma-international.org/ecma-262/#sec-logical-not-operator)
 * [**7.2.13** Abstract Equality Comparison](https://www.ecma-international.org/ecma-262/#sec-abstract-equality-comparison)
 
+## `true` is not equal `![]`, but not equal `[]` too
+Array is not equal `true`, but not Array is not equal `true` too;
+Array is equal `false`, not Array is equal `false` too:
+
+```js 
+true == [] // -> false
+true == ![] // -> false
+
+false == [] // -> true 
+false == ![] // -> true
+```
+### 💡 Explanation:
```

Comment:
Can you write the explanation in more human-readable format? Just cut the items which are not important for that case and keep only the most important.

Keeping things simple is important